packages/discord.js/src/structures/AutoModRule.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+const Base = require('./Base');
+
+/**
+ * Represents an AutoMod rule.
+ */
+class AutoModRule extends Base {
+  constructor(client, data) {
```

Comment:
These props probably need to be patched rather than assigned once. I'd assume when automod updates come through the gateway it would provide update automod rule structures.

Reply:
This behavior has now been added to the docs PR:
> Auto Moderation Rule Update
Sent when a rule is updated. The inner payload is an auto moderation rule object.

Reply:
Should be done with 3c30c9c936f92d530ae179b6d122d4b2f9893eb3.